com.unity.cinemachine.tests/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "com.unity.cinemachine.tests",
+  "type": "tests",
+  "version": "1.0.0",
+  "displayName": "Cinemachine Tests",
+  "dependencies": {
+    "com.unity.test-framework": "1.1.33"
```

Comment:
don't we also depend on cinemachine itself?

Reply:
According to the doc that is not needed but will check.